basis_set_exchange/readers/nwchem.py

```diff
@@ -16,6 +16,11 @@ def _parse_electron_lines(basis_lines, bs_data):
     # They may exist when this is called from other readers
     basis_lines = [x for x in basis_lines if x.lower() != 'end']
 
+    # Basis entry needs to start with 'basis'
+    assert basis_lines[0].lower().find('basis') == 0
```

Comment:
Might be nice to give a better error message

```
if not basis_lines[0].lower().startswith('basis'):
    raise RuntimeError("Basis entry must start with 'basis'")
```

Reply:
Done